bin.src/makeLsstCameraRepository.py

```diff
@@ -258,6 +259,13 @@ def makeDetectorConfigs(detectorLayoutFile, phosimVersion):
             detectorConfigs.append(detConfig)
     return detectorConfigs
 
+
+def getPhosimVersion(defaultDataDir):
+    """Return the phosim version from data/phosim_version.txt"""
+    with open(os.path.join(defaultDataDir, 'phosim_version.txt')) as infile:
+        return infile.read().strip()
```

Comment:
I think phosim produces a version number in the headers.  It may be worth trying to get it from there.